sdk/storage/azure-storage-queues/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 12.0.0-beta.5 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 12.0.0 (2022-04-06)
```

Comment:
Since this is the first GA release of Queues (I think), it might be useful to give a quick highlight/summary of the client API and the header to include as part of "Features Added". That would be helpful for customers to get a pointer of getting started. This content from the changelog also makes it way to the release notes, so its valuable to mention a bullet with some intro-level information